x-pack/plugins/cloud_integrations/cloud_chat/server/plugin.ts

```diff
@@ -42,26 +37,15 @@ export class CloudChatPlugin implements Plugin<void, void, CloudChatSetupDeps, C
         trialBuffer,
         isDev: this.isDev,
         getChatVariant: () =>
-          core.getStartServices().then(([_, { cloudExperiments }]) => {
-            if (!cloudExperiments) {
-              return 'header';
-            } else {
-              return cloudExperiments
-                .getVariation<ChatVariant>('cloud-chat.chat-variant', 'header')
-                .catch(() => 'header');
-            }
-          }),
+          core
+            .getStartServices()
+            .then(([{ featureFlags }]) =>
+              featureFlags.getStringValue<ChatVariant>('cloud-chat.chat-variant', 'header')
+            ),
```

Comment:
Soo much cleaner

Reply:
great point! Addressed in [`9c97aaa` (#188562)](https://github.com/elastic/kibana/pull/188562/commits/9c97aaa0d66ea8effc617693b5a00e46b753eaad)